--- netaddr-0.7.19/setup.py.orig	2017-01-12 00:42:52.000000000 +0100
+++ netaddr-0.7.19/setup.py	2021-03-06 12:37:29.685301598 +0100
@@ -174,11 +174,6 @@
         scripts=['netaddr/tools/netaddr'],
         url='https://github.com/drkjam/netaddr/',
         version=netaddr.__version__,
-        options={
-            'build_scripts': {
-                'executable': '/usr/bin/env python',
-            },
-        },
     )
 
     setup(**setup_options)
